samples/subsys/ipc/ipm_imx/sample.yaml

```diff
@@ -3,7 +3,7 @@ sample:
         using the IPM and transmits it back unchanged
     name: i.MX IPM sample
 tests:
-    sample.subsys.ipc.ipm_imx:
+    sample.ipc.ipm_imx:
```

Comment:
If you named test below `sample.ipc.ipm.ipm_mhu_dual_core:`, that test should be named like `sample.ipc.ipm.imx:`